### DISCUSSION TITLE
童年後悔沒做的事

### DISCUSSION THREAD

**Discussion by thz0000** [1101-GS4719-C/Python-Community-GS4719-C#1895] [General]
很好奇大家的童年都在做什麼？學才藝？幫忙家裡做事？還是泡在幼稚園補習班？我的話，純純的在家，當個小廢物。

也因為這樣，我爸媽最常做的一件事就是帶我去圖書館借書回家(很殺時間XD)，從剛開始的漫畫、兒童書，到後來的偵探推理、武俠修練、言情小說，是我最愛的三種類別，從此培養出閱讀的好習慣(其實是上課都在偷看小說)，每去到一間學校，都很期待他們圖書館的藏書。

但長大之後總會覺得有點小遺憾，周遭好多人都有一技之長，跳舞、樂器、書法、繪畫等等，而自己沒有特別的愛好興趣。雖然我覺得自己已經很幸運、很幸福了(而且閱讀習慣是真的很難培養，所以很多人頭痛的國文是我的愛科哈哈哈)，但有時候會覺得有一技在身的人好像會發光XD

大家有什麼如果重來一遍，想要在童年去做的事嗎？

**Comment by yuchien-lin** [comment]
我當時最遺憾的就是我鋼琴學到一半就放棄了，如果能再重來一次，我希望我這次能堅持下去把鋼琴學好，因為這樣不僅多一項才藝，還能藉由學習彈鋼琴來培養我的耐心。

**Comment by BeaGle0928** [comment]
我覺得小時候總想著趕快長大，等長大了以後可以幹嘛幹嘛等等，但長大了之後才會發現當小孩的好，所以我想我最後悔童年沒做的事就是沒有好好的享受當小孩子的時候吧。但現在回過頭來看，現在的小孩子的生活好像也少了許多樂趣，都充斥著手機等科技產品，這讓我覺得我生在一個恰到好處的時間。

**Comment by chuanmin0317** [comment]
小時候，家人希望我去學樂器，不過當時覺得麻煩就沒有答應，現在希望學吉他但是才發現學費很貴，所以只能夠自學，現在我覺得能夠從小就培養自己的興趣是一件很好的事，像現在我沒事的時候就會彈吉他，不會再一直滑手機浪費時間

**Comment by woohsin** [comment]
好好找個樂器來學，小時候因為定性不足，學甚麼都三分鐘熱度，所以到現在都沒什麼特殊能力(?

**Comment by benbebobo149** [comment]
如果有機會回去告訴過去的自己的話，我蠻想要跟他說對一件事情堅持就好了，不要甚麼都想要達成。三心二意，只會讓你所有的事情都做不好而已。我認為這是一個非常珍貴的教訓，因為我也曾經陷入過這樣的困境。堅持和專注是成功的關鍵，當我們專注於一件事情，才能夠投入更多的時間和精力，從而取得更好的成果。所以，我想告訴過去的自己，要有耐心，堅持追求目標，不要被外界的干擾所影響。最後，我要感謝同學的分享，讓我有機會反思自己的過去經歷，並從中學習到寶貴的教訓。

**Comment by 20050201Shawn** [comment]
如果能重來一遍我會想好好學鋼琴，以前比較叛逆，覺得古典樂跟拜耳很無聊，老師出的曲子作業都上課前幾個小時才開始練，有時候一首曲子可能練一個月才練好。時隔十一年，現在退化到根本不會看五線譜，也覺得對古典樂沒以前那麼排斥，有點後悔沒好好培養樂理吧，但自從七年前退學後開始接觸自己比較有興趣的流行樂，上網學別人彈法那段時光蠻開心的(但現在只會看簡譜)。

**Comment by jeff110303549** [comment]
看到這篇文章我想起好像以前聽說過一個關於小孩子學習力、可塑性最好的說法，至少我看自己現在的能力也蠻多都是源自於童年父母的栽培，不過這並不代表現在開始學習一門技藝太晚，我也有很多技能是在高中乃至大學在參與社團中培養出來的，只要肯付出時間、努力，相信大家也都能夠成為自己夢想中的自己！

**Comment by YANRONG0123** [comment]
我有點後悔童年沒有好好地栽培自己的額外技能，所以現在才會感覺那麼糜爛，追趕別人的進度都相當的困難，除了讀書以外，感覺什麼事情做了也贏不了別人，一個惡性循環的概念。

**Comment by sunset513** [comment]
我後悔童年沒有好好地培養自己的性格，所以造就了我現在在學習上的效率低落。我是個容易分心的人，小時候當我在寫功課或讀書的時候，總是無法靜下心來，總是想著休閒娛樂的事情。這樣的壞習慣，導致我無法良好的吸收知識，使我感覺到我和大家的差距越來越大。不只是在讀書上，在音樂或是體育上，我也覺得別人的學習速度總是比我還要快。

希望在將來的我，能夠慢慢一點一點改善這個壞習慣。最後我要感謝版主拋出這個看似平常，但是具有深度的議題。

**Comment by wanderer657** [comment]
在美國的一個寧靜小鎮上，有一個神秘的圖書館，傳說裡面藏著一本名為《童年的秘密》的古老書籍。這本書據說可以讓人們重返童年，改變他們的過去。

主人公莉莉是一個對童年充滿遺憾的女孩，她在成年後常常懷念著那段單純的時光。有一天，她聽說了有關《童年的秘密》的傳說，決定前往圖書館尋找這本書。

然而，當她翻開書的時候，一股強烈的黑暗力量從書中湧出，將她帶入了一個扭曲而恐怖的童年世界。在這個世界裡，一切都充滿了詭異和恐懼，莉莉不斷遭遇著各種奇怪的事件和怪物的襲擊。

她試圖逃離這個恐怖的童年世界，但卻發現自己無法脫身。最後，她意識到只有面對她的過去，才能解開這個恐怖的詛咒。她勇敢地面對了自己的恐懼和遺憾，最終成功逃出了這個恐怖的童年世界。

莉莉回到現實後，意識到其實她已經擁有了一個美好的生活，過去的遺憾只是她內心的一個陰影。她決定珍惜現在的一切，不再追悔過去，因為只有活在當下，才能擁有真正的幸福。

**Comment by thomasxshen** [comment]
我後悔在童年的時候沒有好好學習羽球，小時候我就很喜歡打羽球，憑藉著勤奮的練習與濃厚的興趣，我很快就進到了羽球班。在羽球班中，我也算是佼佼者，有著不俗的實力被教練另眼相看，因此教練也對我特殊關照，還記得當時只要沒有把球發進外角，就得跑五圈羽球場，架勢不正確的時候也會被女教練嚴厲指責，或許當時教練只是對我寄與厚望，希望我能夠成為班級的標竿，但這對於還是小學生的我是個不小的心理壓力，總是心想我都這麼認真在學了，你就不能誇獎我一下嗎？還老是罵我罰我？來年我便退出了羽球班，從此陷入長達六年的徘徊，除了唸書什麼都不敢興趣，也瞧不上那些不唸書不幹正事的人，一直抱持著這種自卑型優越感直至大學才得到救贖。

**Comment by 155Julia** [comment]
童年時我有一些後悔沒有做的事情，其中最大的後悔之一是我沒有更認真地讀書，另外也沒有專注地培養一項樂器，我學習了許多像是鋼琴、打鼓、小提琴、木吉他、電吉他，但從未真正專注並精通其中任何一項。如果可以重來，我希望除了能夠專心地投入到學習中，才藝方面專注在小提琴跟木吉他就好。透過堅持不懈地努力，才能培養出更深厚的技能和專業知識。

**Comment by SusuTima** [comment]
我反而比較後悔童年的時候沒怎麼去看卡通，現在偶爾聽同學聊天，連卡通的角色長怎樣都不知道。小時候爸媽不准看，現在長大也沒什麼興趣了。

**Comment by 710388** [comment]
同年後悔沒有好好學樂器，看其他人都從小到大練習一個樂器，大學的時候還可以出去比賽，好羨慕，這種東西感覺現在才開始練太慢了。

**Comment by LinChloe** [comment]
後悔以前太聽爸媽的話，沒有多嘗試點東西。
等到現在學業繁忙反而少了很多機會。
我覺得人生在世要多做點事情，不要瞻前顧後、怕東怕西的反而失去更多。

**Comment by Mitochondrionjessica** [comment]
後悔童年不懂得享受與珍惜童年，一心想要快快長大，覺得長大有自由去做喜歡跟想要做的事情。殊不知幻想很美好，現實很骨感，長大確實自由，但相對的也有很大的責任，無暇也無閒無錢去做小時候小要做的事情。更可怕的是長大會有很多後顧之憂，會害怕現在做的決定未來承擔不起。說了這麼多，還是好好珍惜現在活在當下吧~

**Comment by willychiuchiu** [comment]
跟樓上大部分的人一樣，我真的很後悔沒有多學一點音樂，每次看到同學彈鋼琴、彈吉他都超帥⋯尤其拿著一把吉他自彈自唱，真的超羨慕（超好把妹），長大以後就沒有太多的時間能去接觸，而且學費也要好貴好貴，看來只能等未來有錢有閒的時候才有機會去圓小時候的夢了

**Comment by 110101501** [comment]
蠻後悔小時候沒有好好學琴。起初小時候有在學，但學到後來就放棄不想練了（不知道自己哪根筋不對）。長大後想學琴就不如過去小時候學習力強的時刻，也耗時耗金錢。回想起來感到頗為遺憾。

**Comment by happy246810** [comment]
我覺得我的童年算是活得挺快樂的，有家人陪伴，規律的作息，但如果真要說有什麼後悔的話，應該就是沒有好好珍惜那段天真無邪的童年，那時總想著要快點長大，騎車，出去玩等。

**Comment by yxin214** [comment]
我覺得是沒有好好學習一項才藝，雖然學過音樂，但都沒有持續練習，好想趁寒暑假來精進自己啊！

**Comment by yyujimmy** [comment]
學習樂器或舞蹈：許多人希望自己能夠精通某種樂器或舞蹈，這不僅可以培養音樂或舞蹈方面的才藝，還可以提升自信心和表達能力。
藝術創作：有些人對繪畫、書法、手工藝等藝術創作感興趣，希望能夠在童年時期培養這些技能，並將其發展成為自己的興趣或職業。
運動和健身：對於喜歡運動的人來說，童年是一個很好的時機來學習各種運動技能，例如游泳、籃球、足球等，這不僅有助於身體健康，還可以培養團隊合作和領導能力。
學習新語言或技術：在現今全球化的時代，掌握多種語言或者學習新的技術對於個人的發展至關重要，因此一些人可能希望在童年時期就開始學習新的語言或技術。
參與社會活動：在童年時期參與社會活動或志願服務可以培養個人的社會責任感和公民意識，同時也有助於擴展人際關係網絡和增加社交技巧。

**Comment by KH1126** [comment]
我比較後悔從高中後因為課業的關係沒有繼續學習大提琴。
現在太久沒有拉大提琴，蠻多東西都忘記了。

**Comment by YOGA012** [comment]
國三因為會考衝刺要多上一節課而放棄原本每天放學都會參加的路跑隊訓練，因為是自由參加所以不是每個人都多上一節，看著同屆的同學一個個持續進步，比賽都能上台領獎再看看我課業成績幾乎沒甚麼起色而且體能也退化到無法跟上隊員的步伐，就覺得當初很後悔參加會考衝刺班。

**Comment by William08290829** [comment]
我後悔的是沒有多去參加學校的活動多多認識人，因為我意識到社交網絡對於個人成長和未來發展的重要性。在學校的時候，我可能更多地專注於學業，而忽略了與同學們的交流和互動。這樣一來，我錯過了許多與人相處、團隊合作以及社交技巧方面的寶貴機會。現在回想起來，我意識到在學校的社交活動中建立的人際關係不僅對於學業有助益，而且也對於個人成長和未來職場發展至關重要。所以，如果我能夠重新來過，我會更加積極地參與學校的各種活動，努力擴大自己的社交圈子，以便在未來的人生道路上能夠更加順利地前進。

**Comment by jim1743** [comment]
我最後悔沒在以前定時運動以及充足的睡眠，因為我現在的身高處於170這種不高不矮的地方，每次走在路上遇到身高高的人都很羨慕，雖然身高不高這件事從沒帶給我任何負面影響，但若能回到過去，我一定定期運動，好讓自己身高再高一點。

**Comment by cassy0201** [comment]
希望小時候的自己能多嘗試一些不同的興趣愛好，多睡覺

**Comment by tshsu04224** [comment]
我後悔幼稚園之後沒有繼續學鋼琴，想當初幼稚園還曾經參加過音樂會，跟我的鋼琴老師一起表演四手聯彈，結果升上小學後沒有繼續學，鋼琴技能就被我忘得一乾二淨了，現在連樂譜上的符號都看不懂，只記得五線譜跟do re no fa so，現在一個樂器也不會真的是我最後悔的事

**Comment by yubo0oo** [comment]
如果有機會的話
我會繼續學爵士鼓
還會答應當年喜歡我的女同學
她現在超正

**Comment by YeeDaniel** [comment]
童年哦，我覺得多少有遺憾欸，對我來說應該是繼續參加樂隊吧，當初在國三如果繼續參加樂隊，不僅可以去國外表演，也可以維持這段不容易的情誼。而且當初在樂隊也有心儀的女生，或許也會因此成了吧。

**Comment by JustAHedgehog** [comment]
如果是童年最後悔沒做的事情，我個人是沒有養成一項運動的習慣，雖然甚麼運動都會一些，但沒有一項真的十分擅長，雖然在大學期間有刻意學習，然有些身體習慣很難改變，所以仍在克服中><

**Comment by OHoln** [comment]
應該是後悔沒有多運動吧，沒有趁著小時候時間多的時候培養基礎身體能力，現在就算對各種運動項目有興趣也總是力不從心，實在挺可惜的。
